content/sensu-go/5.18/sensuctl/reference.md

```diff
@@ -666,30 +696,47 @@ For information about the operators and fields you can use in response filters,
 | `notin`  | Not included in | `slack notin check.handlers`
 | `&&`     | Logical AND     | `check.publish == true && slack in check.handlers`
 
-### Filter responses with labels
+For details about operators, see [API response filtering operators][47].
+
+### Filter responses with label selectors
 
 Use the `--label-selector` flag to filter responses using custom labels.
 
-In this example, the command returns entities with the `proxy_type` label set to `switch`:
+For example, to return entities with the `proxy_type` label set to `switch`:
 
 {{< highlight shell >}}
 sensuctl entity list --label-selector 'proxy_type == switch'
 {{< /highlight >}}
 
-### Filter responses with resource attributes
+### Filter responses with field selectors
 
-Use the `--field-selector` flag to filter responses using selected resource attributes.
-To see the resource attributes you can use in response filter statements, see the [API docs][29].
+Use the `--field-selector` flag to filter responses using specific [resource attributes][29].
 
-In this example, the command returns entities with the `switches` subscription:
+For example, to return entities with the `switches` subscription:
 
 {{< highlight shell >}}
 sensuctl entity list --field-selector 'switches in entity.subscriptions'
 {{< /highlight >}}
 
-You can also combine the `--label-selector` and `--field-selector` flags.
+To retrieve all events at `2` (CRITICAL) status:
```

Comment:
This sounds weird to me. It might sound better with something like `all (CRITICAL) '2' event status` or `all events equaling status '2' (CRITICAL)`

Reply:
I see what you mean. I will fix it now.